packages/tx/test/inputValue.spec.ts

```diff
@@ -0,0 +1,134 @@
+import tape from 'tape'
+import { Address, AddressLike, BN, BNLike, BufferLike, toBuffer } from 'ethereumjs-util'
+import Common, { Chain, Hardfork } from '@ethereumjs/common'
+import { Transaction } from '../src'
+
+// @returns: Array with subtypes of the AddressLike type for a given address
+function generateAddressLikeValues(address: string): AddressLike[] {
+  return [address, toBuffer(address), new Address(toBuffer(address))]
+}
+
+// @returns: Array with subtypes of the BNLike type for a given number
+function generateBNLikeValues(value: number): BNLike[] {
+  return [value, new BN(value), `0x${value.toString(16)}`, toBuffer(value)]
+}
+
+// @returns: Array with subtypes of the BufferLike type for a given string
+function generateBufferLikeValues(value: string): BufferLike[] {
+  return [value, toBuffer(value)]
+}
+
+interface GenerateCombinationsArgs {
+  options: { [x: string]: any }
+  optionIndex?: number
+  results?: { [x: string]: any }[]
+  current?: { [x: string]: any }
+}
+
+export function generateCombinations({
+  options,
+  optionIndex = 0,
+  results = [],
+  current = {},
+}: GenerateCombinationsArgs) {
+  const allKeys = Object.keys(options)
+  const optionKey = allKeys[optionIndex]
+
+  const values = options[optionKey]
+
+  for (let i = 0; i < values.length; i++) {
+    current[optionKey] = values[i]
+
+    if (optionIndex + 1 < allKeys.length) {
+      generateCombinations({ options, optionIndex: optionIndex + 1, results, current })
+    } else {
+      results.push(current)
+    }
+  }
+
+  return results
+}
+
+// Deterministic pseudorandom number generator
+function mulberry32(seed: number) {
+  let t = (seed += 0x6d2b79f5)
+  t = Math.imul(t ^ (t >>> 15), t | 1)
+  t ^= t + Math.imul(t ^ (t >>> 7), t | 61)
+  return ((t ^ (t >>> 14)) >>> 0) / 4294967296
+}
+
+function getRandomSubarray(array: any[], size: number) {
+  const shuffled = array.slice(0),
+    seed = 1559
```

Comment:
This is probably just me not being a Javascript guru but this syntax... 
```
 const shuffled = array.slice(0),
    seed = 1
```
threw me for a loop. Is the comma operator here just applying `const` to the `seed = 1` expression?  Reading on mozilla dev docs here didn't make it any more obvious to me - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator

Reply:
Yeah, it's a shorthand way of declaring multiple variables, but I agree it's not too readable and commonly used, I'll revert to an explicit declaration per line.